cmd/inspect.go

```diff
@@ -128,9 +133,8 @@ func printContainerInspect(c *clab.CLab, containers []types.GenericContainer, br
 			IPv4Address: getContainerIPv4(cont, bridgeName),
 			IPv6Address: getContainerIPv6(cont, bridgeName),
 		}
-		if len(cont.ID) > 11 {
```

Comment:
this we need to keep to trim container id to 12 chars

Reply:
as discussed offline, we need to take care of the runtime ... containerd doesn't use UID kind of identifiers but the container names...

Reply:
added in bee3dab